mm2src/coins/utxo/utxo_common.rs

```diff
@@ -3763,6 +3932,11 @@ where
         FeeApproxStage::WithoutApprox => return dynamic_fee,
         // Take into account that the dynamic fee may increase during the swap by [`UtxoCoinFields::tx_fee_volatility_percent`].
         FeeApproxStage::StartSwap => base_percent,
+        // Take into account that the dynamic fee may increase after roughly the locktime is expired [`UtxoCoinFields::tx_fee_volatility_percent`]:
+        // - watcher can refund the taker payment after the locktime + an extra time to wait for the takers
+        // - the watcher can spend the taker_spends_maker_payment right after locktime/2, but the worst case should be considered which is slightly before
+        //   the locktime is expired
+        FeeApproxStage::WatcherPreimage => base_percent, //This needs discussion
```

Comment:
It needs to discuss, how to manage percent for `WatcherPreimage` depending on different situations.
But why we just have constant percent 3 on watcher preimage for eth?
Is it because the behavior is already defined in the [function increase_by_percent_one_gwei](https://github.com/KomodoPlatform/atomicDEX-API/blob/watchtower/mm2src/coins/eth.rs#L3800) and there is no need to consider other cases?

Reply:
When we talked with Artem we decided to set the fee increase for the watcher preimages the same as in the `FeeApproxStage::StartSwap` case, so I also did the same for Ethereum. Actually at this point it isn't that important because the watchers don't even work for Ethereum yet and we can also decide to set the fee increase rate to something different later.

Reply:
Ok, got it, smth will be later. then will close this question.